testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/elytron/SlaveHostControllerElytronAuthenticationTestCase.java

```diff
@@ -118,13 +118,15 @@ private void slaveWithPlainMechanism() throws Exception {
 
         // Reload the slave and check that it produces a registered slave
         reloadSlave();
+        testSupport.getDomainSlaveLifecycleUtil().awaitHostController(System.currentTimeMillis());
         readHostControllerStatus(getDomainMasterClient());
 
         // Set the allowed mechanism back to Digest-MD5
         getDomainMasterClient().execute(changePresentedMechanisms("master", new HashSet<>(Arrays.asList("DIGEST-MD5"))));
         getDomainSlaveClient().execute(changeSaslMechanism("slave", "DIGEST-MD5"));
         reloadSlave();
         testSupport.getDomainSlaveLifecycleUtil().awaitHostController(System.currentTimeMillis());
+        assertTrue("Host \"slave\" did not reconnect to master with plain mechanism.", hostRunning(FAILED_RELOAD_TIMEOUT_MILLIS));
         readHostControllerStatus(domainMasterClient);
```

Comment:
L129 makes L130 redundant.

This is OT to this PR but the other use of hostRunning in this class kind of sucks. AIUI it basically means the test will wait around for 10 secs waiting for the slave to *not* reauth. We need to be more clever.